install_config/install/docker_registry.adoc

```diff
@@ -173,20 +173,101 @@ $ oc logs docker-registry-1-da73t
 
 == File Storage
 
-Tag and image metadata is stored in OpenShift, but the registry owns layer and
-signature data. The registry stores data in a volume that is mounted into the
-registry container at *_/registry_*. You can
-link:../../dev_guide/executing_remote_commands.html[execute a remote command] in
-the registry container to see the contents:
+Tag and image metadata is stored in OpenShift, but the registry stores layer and
+signature data in a volume that is mounted into the registry container at
+*_/registry_*. As `oc exec` does not work on privileged containers, to find the
+correct node housing the container manually SSH into the node housing the
+container and then run `docker exec` on the pod.
 
-[NOTE]
+To see a registry's contents, SSH into the node, then list the contents:
```

Comment:
If a form of the suggestion in the above comment (line 180) is used, then this line could be removed (would become redundant as well).